src/account/operations/address_generation.rs

```diff
@@ -18,14 +18,16 @@ use crate::events::types::{AddressData, WalletEvent};
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct AddressGenerationOptions {
     pub internal: bool,
-    pub metadata: GenerateAddressMetadata,
+    pub metadata: GenerateAddressOptions,
```

Comment:
Should we rename this to `options` as well ?

Reply:
That's done in https://github.com/iotaledger/wallet.rs/pull/1516